src/main/java/org/opentripplanner/middleware/controllers/api/ApiController.java

```diff
@@ -334,7 +353,9 @@ private T createOrUpdate(Request req, Response res) {
                 }
                 // Check that requesting user can manage entity.
                 if (!preExistingObject.canBeManagedBy(requestingUser)) {
```

Comment:
Need to check that the requesting user can also manage the user id assigned to the entity to update, otherwise there is a possibility that an `ApiUser` can assign the an entity (e.g. a `MonitoredTrip`) to a user not managed by that `ApiUser`... I thought I saw a check for that somewhere and can't find it.

Reply:
@binh-dam-ibigroup I think this is already covered in canBeManagedBy. If the requesting user is updating a `MonitoredTrip` a call will be made to `MonitoredTrip.canBeManagedBy` where a check is made to confirm the `OtpUser` is associated with the requesting `ApiUser`.

Reply:
@br648 correct, however `canBeManagedBy` doesn't check the `userId` field in the submitted data, does it?

Reply:
I will let @landonreed  and @evansiroky weigh in on how we want to proceed with this. I don't feel too strongly about it.

Reply:
The checking of submitted data will be carried out under a new PR. The issue has been logged here: https://github.com/ibi-group/otp-middleware/issues/94